doc/api.yml

```diff
@@ -3275,6 +3275,163 @@ paths:
         '500':
           $ref: '#/components/responses/500'
 
+  '/letterbox':
+    get:
+      tags:
+        - Letterbox
+      summary: All deposit in the letterbox
+      operationId: get all deposit in the letterbox
+      parameters:
+        - name: page
+          in: query
+          schema:
+            $ref: '#/components/schemas/Page'
+          description: Set value to 1 by default if null is provided
+          required: false
+        - name: page_size
+          in: query
+          description: Set value to 15 by default if null is provided
+          schema:
+            $ref: '#/components/schemas/PageSize'
+          required: false
+        - name: student_ref
+          in: query
+          description: Filter by student ref
+          schema:
+            type: string
+          required: false
+      responses:
+        '200':
+          description: All deposit in the global letterbox
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Deposit'
+        '400':
+          $ref: '#/components/responses/400'
+        '403':
+          $ref: '#/components/responses/403'
+        '404':
+          $ref: '#/components/responses/404'
+        '429':
+          $ref: '#/components/responses/429'
+        '500':
+          $ref: '#/components/responses/500'
+    put:
+      tags:
+        - Letterbox
+      summary: Update deposits status
+      operationId: updateDespositsStatus
```

Comment:
update depositStatus should have a status in its requestbody, otherwise it simply means that you've read the deposit and the operation would be readDeposit